### ISSUE TITLE
Title attribute on icons

### ISSUE THREAD

**Issue by QuantumInformation** [react-icons/react-icons#228]
It would be nice to do something like this:

                      <FaExclamationCircle  title="Site has unsaved changes" />

**Comment by paustria** [comment]
Was proposed here but didn't get merge. I'm not sure why. https://github.com/react-icons/react-icons/issues/110

**Comment by kamijin-fanta** [comment]
Looks like it works. sample: https://codesandbox.io/s/0xlxzp9lmn

If you use TypeScript, it will be the effect of type definition. 
 https://github.com/DefinitelyTyped/DefinitelyTyped/blob/584682b746557368e8adb3339ac973bd7da03740/types/react/index.d.ts#L2208-L2472

Does this code work?
```
<FaFolder {...{title: "folder"}} />
```

**Comment by Slapbox** [comment]
@kamijin-fanta what browser did you test in? The sample doesn't work for me in Chrome or Firefox on Windows.

**Comment by paustria** [comment]
@kamijin-fanta To make svg complying with accessibility, the `title` tag has to be a child of the `svg` tag.
In your example, it has to be
```
<svg stroke="currentColor" fill="currentColor" stroke-width="0" viewBox="0 0 512 512" title="This is folder" height="2.5em" width="2.5em" xmlns="http://www.w3.org/2000/svg">
     <title>This is folder</title>
     <path d="M464 128H272l-64-64H48C21.49 64 0 85.49 0 112v288c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V176c0-26.51-21.49-48-48-48z"></path>
</svg>
```

The reader won't read prop `title` on `svg`. I tested based on this tool. https://www.nvaccess.org/.

My current work around is I'm using `material-ui` icons with `titleAccess` props. https://material-ui.com/style/icons/#semantic-svg-icons

https://github.com/react-icons/react-icons/issues/110 should resolve this issue. Please consider.

**Comment by kamijin-fanta** [comment]
@paustria  I did not recognize the problem correctly. I think this should be supported. thanks.

**Comment by paustria** [comment]
@kamijin-fanta I created an above PR for this since there is no one working on it yet. Please code review if it looks OK for you. Please also update `README.md` if it gets merged. Since it's in `.gitignore`, I can't update it in this PR. Thank you for considering.

**Comment by paustria** [comment]
@kamijin-fanta Is there anyway we can get this PR https://github.com/react-icons/react-icons/pull/231 merged? I haven't finished migrate my projects to `material-ui`. If this PR is in, I don't need to migrate them. Please let me know. Thank you!

**Comment by YannDuv** [comment]
Is there any progression on the validation of this merge request please ? ⌛️ 💥

**Comment by paustria** [comment]
@YannDuv This should be in 3.7.0. https://github.com/react-icons/react-icons/pull/231 It was merged 4 days ago. I think @kamijin-fanta  forget to tag it. I just update it and see the changes now. Please let us know if it doesn't work.

@kamijin-fanta Thank you for fixing and merging. Could you please close this ticket?

**Comment by YannDuv** [comment]
And it works like a charm. Thank you 🙇

**Comment by kamijin-fanta** [comment]
sorry.

this issue fix in v3.7.0.
https://github.com/react-icons/react-icons/releases/tag/v3.7.0
